lib/ansible/module_utils/api.py

```diff
@@ -107,11 +107,11 @@ def wrapper(f):
         retry_count = 0
 
         def retried(*args, **kwargs):
+            global retry_count
```

Comment:
This is an incorrect change.  `retry_count` is not a global.  Technically it is a nonlocal.  `nonlocal` is not available in py2, so this closure just needs rewritten